client/api/filesystem/v1beta2/api.proto

```diff
@@ -100,12 +78,8 @@ message RmdirRequest {
     // Maximum path length will be capped to 260 characters.
     string path = 1;
 
-    // Context of the path parameter.
-    // This is used to validate prefix for absolute paths passed
-    PathContext context = 2;
-
     // Force remove all contents under path (if any).
-    bool force = 3;
+    bool force = 2;
```

Comment:
Somehow the comment here got deleted. Can we use `reserved` https://developers.google.com/protocol-buffers/docs/proto3#reserved here to prevent the re-numbering?

Reply:
I think this is useful if the same proto file is used across versions, because we have new protos for every version I'm not sure if this makes sense in this project